tools/github/Add-AzsdkMilestones.ps1

```diff
@@ -2,7 +2,7 @@
 param (
     [Parameter(ParameterSetName = 'RepositoryFile')]
     [ValidateScript({Test-Path $_ -PathType 'Leaf'})]
-    [string]$RepositoryFilePath = "./repositories.txt",
+    [string]$RepositoryFilePath = '$PSScriptRoot/repositories.txt',
```

Comment:
This won't work. Needs to be double quotes to resolve variables.

Reply:
.... I did not know this.   Thanks!